### PR TITLE
pkg/config(cdc): add MessagesConfig for configuring peer-messages

### PR DESCRIPTION
**Pull request by liuzix**
### What problem does this PR solve? 
- We need some user-configurable parameters passed to `MessageServer` and `MessageClient` in `pkg/p2p`.

### What is changed and how it works?
- Added a section `messages` in the server configuration file.

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has persistent data change

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by sdojjy — APPROVED** [review]

File: pkg/config/server_config.go [hunk 1]
```diff
@@ -146,6 +147,7 @@ type ServerConfig struct {
 	PerTableMemoryQuota uint64          `toml:"per-table-memory-quota" json:"per-table-memory-quota"`
 	KVClient            *KVClientConfig `toml:"kv-client" json:"kv-client"`
 	Debug               *DebugConfig    `toml:"debug" json:"debug"`
+	Messages            *MessagesConfig `toml:"messages" json:"messages"`
```
Comment (overvenus) [inline pkg/config/server_config.go:150]: Before GA, could you move it into `Debug`?

**Review by overvenus — COMMENTED** [review]
Rest LGTM

File: pkg/config/messages.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+	ServerWorkerPoolSize         int          `toml:"server-worker-pool-size" json:"server-worker-pool-size"`
+}
+
+// read only
+var defaultMessageConfig = &MessagesConfig{
+	ClientMaxBatchInterval:       TomlDuration(time.Millisecond * 100),
+	ClientMaxBatchSize:           8 * 1024, // 8MB
+	ClientMaxBatchCount:          128,
+	ClientRetryRateLimit:         1.0, // Once per second
+	ServerMaxPendingMessageCount: 102400,
```
Comment (3AceShowHand) [inline pkg/config/messages.go:41]: Is `102400` a reasonable default value ?

Comment (liuzix) [reply pkg/config/messages.go:41]: I think it is:
- MessageServer is a singleton.
- Individual messages are small. Large messages would be extremely rare.

**Review by 3AceShowHand — APPROVED** [review]

**Review by overvenus — APPROVED** [review]

**Pull request merged**